### ISSUE TITLE
Navigation link hover color has bad contrast with background color

### ISSUE THREAD

**Issue by jseppi** [18F/federalist#1141]
This might just be happening on Windows Chrome, I'm not sure. Will double check later on my work MBP.

![nav-hover](https://user-images.githubusercontent.com/697848/29996701-52794d86-8fc9-11e7-9b9d-811e5f0e989a.gif)

**Comment by line47** [comment]
I think this issue can been resolved? @wslack

**Comment by jseppi** [comment]
Get Help is still weirdly the wrong color on hover: 

<img width="245" alt="screen shot 2017-11-30 at 11 02 41 am" src="https://user-images.githubusercontent.com/697848/33443729-e88ec186-d5bd-11e7-8119-f477887a428d.png">

**Comment by jseppi** [comment]
Ah, I think the problem is the `:hover` and `:visited` rules aren't quite aligned. If you open the site in a new browser or incognito window, all the links on the home page have the bad-contrast dark blue color on hover.

**Comment by line47** [comment]
Ah! thanks @jseppi I missed that nuance 👍

**Comment by line47** [comment]
https://github.com/18F/federalist/pull/1349 to address the issue
